web/src/beta/features/ProjectSettings/innerPages/PluginSettings/hooks.ts

```diff
@@ -0,0 +1,186 @@
+import { useApolloClient } from "@apollo/client";
+import { useCallback, useMemo } from "react";
+
+// TODO: migrate to services API
+import {
+  useGetInstalledPluginsQuery,
+  useInstallPluginMutation,
+  useUninstallPluginMutation,
+  useUploadPluginMutation,
+  useUpgradePluginMutation,
+} from "@reearth/classic/gql/graphql-client-api";
+import { useT, useLang } from "@reearth/services/i18n";
+import { useNotification } from "@reearth/services/state";
+
+export type Plugin = {
+  fullId: string;
```

Comment:
I can't really see where you are using this. Also since it is basically just id + version I don't know we need this either way.